src/sentry/static/sentry/app/components/gridEditable/index.tsx

```diff
@@ -333,24 +274,82 @@ class GridEditable<
     );
   };
 
+  renderError = () => {
+    const {error} = this.props;
+
+    return (
+      <GridBody>
```

Comment:
These seem to be repeated a few times, is it possible to move them up to the call site?

Reply:
Good catch.